doc/interpolation.rst

```diff
@@ -48,6 +48,23 @@ array-like, which gives the interpolated result as an array.
     # interpolation
     da.interp(time=[2.5, 3.5])
 
+
+To interpolate data with a :py:func:`numpy.datetime64` coordinate you have to 
+wrap it explicitly in a :py:func:`numpy.datetime64` object.
```

Comment:
It is currently true, but I think we could support `da_dt64.interp(time='2000-01-02')` in the future.
Do you mind to send another PR to do this?

Reply:
I'm trying to wrap up another PR. Unfortunately, I don't want to over promise and may not be able to get to this until a couple of weeks. If you have ideas of how to do this we can move the discussion to https://github.com/pydata/xarray/issues/2284